app/views/campaigns/show.html.erb

```diff
@@ -88,7 +88,7 @@
 
     <% if !@campaign.child_campaign? %>
       <aside class='u-marginBottom--15 pastelBox--grey'>
-        <a class='button u-width--full' target='_blank' if-branded='background-color, dark' href='<%= @campaign.finished? ? "/peer-to-peer?npo_id=#{@nonprofit.id}" : "/peer-to-peer?campaign_id=#{ @peer_to_peer_campaign_param}" %>'>
+        <a class='button u-width--full' target='_blank' if-branded='background-color, dark' href='<%= @campaign.deadline_has_been_reached? "/peer-to-peer?npo_id=#{@nonprofit.id}" : "/peer-to-peer?campaign_id=#{ @peer_to_peer_campaign_param}" %>'>
```

Comment:
As part of this pull request, since we're messing with it, can you replace these examples of hardcoded paths with rails path helpers? You can find out more about them in the guide on routing at the official Rails website.